lrauv_system_tests/test/vehicle/test_battery_low_charge.cc

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (C) 2022 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+/*
+ * Development of this module has been funded by the Monterey Bay Aquarium
+ * Research Institute (MBARI) and the David and Lucile Packard Foundation
+ */
+
+#include <gtest/gtest.h>
+
+#include <ignition/msgs/battery_state.pb.h>
+
+#include <chrono>
+
+#include <lrauv_ignition_plugins/lrauv_command.pb.h>
+
+#include "lrauv_system_tests/TestFixture.hh"
+
+using namespace ignition;
+
+std::vector<msgs::BatteryState> batteryMsgs;
+
+void recordBatteryMsgs(const msgs::BatteryState &_msg)
+{
+  batteryMsgs.push_back(_msg);
+}
+
+//////////////////////////////////////////////////
+/// Test if the battery discharges with time with the specified
+/// disacharge power rate, when starting with low charge.
```

Comment:
@adityapande-1995 nit: 
```suggestion
/// discharge power rate, when starting with low charge.
```

Reply:
Fixed : https://github.com/osrf/lrauv/pull/207/commits/a4db680cb8d12dc274085c5dc115cb45ac281f8c